src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -298,38 +300,30 @@
 
   The cursor is closed when `func` returns. If an exception is thrown,
   the query is cancelled."
-  [func sql params]
+  [[sql & params] func]
```

Comment:
then it was strange before too. but we should leave it.

On Fri, May 29, 2015 at 3:36 PM, Andrew Roetker <notifications@github.com>
wrote:

> In src/puppetlabs/puppetdb/jdbc.clj
> <https://github.com/puppetlabs/puppetdb/pull/1425#discussion_r31370824>:
>
> > @@ -298,38 +300,30 @@
> >
> >    The cursor is closed when `func` returns. If an exception is thrown,
> >    the query is cancelled."
> > -  [func sql params]
> > +  [[sql & params] func]
>
> It's the same as streamed-query-results was before
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetdb/pull/1425/files#r31370824>.
>